cmake/modules/OpmPythonAddEmbeddedTests.cmake

```diff
@@ -1,16 +1,198 @@
-function (add_opm_embedded_tests)
+
+function (add_opm_embedded_tests_from_opm_common)
   if (Boost_UNIT_TEST_FRAMEWORK_FOUND)
     set(_libs mocksim opmcommon ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY})
 
-    foreach( test test_msim test_msim_ACTIONX test_msim_EXIT)
-      opm_add_test(${test} SOURCES embedded/tests/msim/${test}.cpp
+    foreach( test msim msim_ACTIONX msim_EXIT)
+      set (test_name test_${test})
+      opm_add_test(${test_name} SOURCES embedded/tests/msim/${test_name}.cpp
         LIBRARIES ${_libs}
         WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/embedded/tests
         CONDITION ${HAVE_ECL_INPUT})
-      target_include_directories(${test} PUBLIC ${opm-common_SOURCE_DIR}/msim/include)
+      target_include_directories(${test_name} PUBLIC ${opm-common_SOURCE_DIR}/msim/include)
+      set_tests_properties(${test} PROPERTIES ENVIRONMENT "${PYTHON_PATH}")
     endforeach()
-    set_tests_properties(msim_ACTIONX PROPERTIES
-            ENVIRONMENT "PYTHONPATH=${PROJECT_BINARY_DIR}/opm-python/src:$ENV{PYTHONPATH}")
   endif()
 endfunction()
-add_opm_embedded_tests()
\ No newline at end of file
+
+#
+#  TODO: This function is part of the opm-simulators file compareECLFiles.cmake and should
```

Comment:
Yes, ideally this is done before merging.

Reply:
Yes :)